src/views/Package/components/OperatorArea/OperatorArea.tsx

```diff
@@ -74,6 +81,12 @@ export const OperatorArea: FunctionComponent<OperatorAreaProps> = ({
       items.push(<>Registry: {registryLink}</>);
     }
 
+    items.push(
+      <Link color="blue.500" href="mailto:abuse@amazon.com">
```

Comment:
Maybe prefill the subject? Will make it easy for us to filter those out later.  

```suggestion
      <Link color="blue.500" href="mailto:abuse@amazon.com?subject={`ConstructHub - Report of abusive package: ${package}`}">
```